cryptofeed/exchanges/bitmex.py

```diff
@@ -102,6 +107,48 @@ def parse_order(self, data):
             order['lastTradeTimestamp'] = self.timestamp_normalize(data.get('transactTime'))
         return order
 
+    def parse_position(self, data):
+        timestamp = self.timestamp_normalize(data['timestamp'])
+        position = {
+            'timestamp': timestamp
+        }
+        if data.get('symbol') is not None:
+            position['symbol'] = self.exchange_symbol_to_std_symbol(data['symbol'])
+        if data.get('initMargin') is not None:
+            position['initialMargin'] = data.get('initMargin')
+        if data.get('maintMargin') is not None:
+            position['maintenanceMargin'] = Decimal(data['maintMargin'])
+        if data.get('avgEntryPrice') is not None:
+            position['entryPrice'] = data.get('avgEntryPrice')
+        if data.get('homeNotional') is not None:
+            position['homeNotional'] = data.get('homeNotional')
+        if data.get('foreignNotional') is not None:
+            position['foreignNotional'] = data.get('foreignNotional')
+        if data.get('leverage') is not None:
+            position['leverage'] = data.get('leverage')
+        if data.get('unrealisedPnl') is not None:
+            position['unrealizedPnl'] = Decimal(data['unrealisedPnl'])
+        if data.get('currentQty') is not None:
+            position['contracts'] = data.get('currentQty')
+        if data.get('liquidationPrice') is not None:
+            position['liquidationPrice'] = Decimal(data['liquidationPrice'])
+        if data.get('markPrice') is not None:
+            position['markPrice'] = Decimal(data['markPrice'])
+        return position
```

Comment:
Why does it decimalize some numbers and others are not?
How about converting all numbers to decimal for generalization?